src/IncomingWebhook.ts

```diff
@@ -49,7 +49,10 @@ export class IncomingWebhook {
    * @param message the message (a simple string, or an object describing the message)
    * @param callback
    */
-  public send(message: string | IncomingWebhookSendArguments, callback: IncomingWebhookResultCallback): void {
+  public send(message: string | IncomingWebhookSendArguments): Promise<void>;
```

Comment:
if its going to be a Promise of something, why not a promise of a `string` (the response text), or better yet, define a type for it (an object that might get more properties in the future).

Reply:
While I agree that *something* should be returned, I somewhat feel that's out of the scope of the issue I was originally targeting.

I simply was aiming to expose the [already-existing implementation](https://github.com/slackapi/node-slack-sdk/blob/a206c3e1286be958c963795f9ce80a18cb85ce7d/src/IncomingWebhook.ts#L63-L68) (which already intentionally doesn't return anything) as a `Promise`, rather than change the implementation to be of more aid.

Anyhow, I'll gladly tack this on 👍✨